src/settings.cpp

```diff
@@ -43,6 +43,11 @@ hipBoneSettings_t::hipBoneSettings_t(const int argc, char** argv, comm_t _comm):
             "FALSE",
             "Enable direct access of GPU memory in MPI");
 
+  newToggle("-ver", "--version",
```

Comment:
Personally, I'd prefer
```
newToggle("-V", "--version",
```
since `-v` is already taken by verbosity.  This is consistent with other tools' behaviour.